schemachange/sc_records.c

```diff
@@ -2713,7 +2713,7 @@ static int live_sc_redo_update(struct convert_record_data *data, DB_LOGC *logc,
 
     if (!data->s->sc_convert_done[rec->dtastripe] &&
         is_genid_right_of_stripe_pointer(data->to->handle, genid,
```

Comment:
Should we have two functions, one that takes the stripe ptr value just like before, and the new one that figures out the stripe from genid?

Reply:
I thought about that.. but the stripe info from genid should be the same as the one passed in. else we have other issues